### PR TITLE
TK-441 Check attendees availability

### PR DESCRIPTION
**Pull request by jb-lunatech**
This ticket was marked as blocked because events store _start_ and _end_ as `LocalDateTime` rather than _date_ and _duration_.
As a consequence, the test of this feature is disables because the SQL query relies on `date()` function that is not supported by H2. With the change to _date_ and _duration_, the comparison of same date will not require any truncation/ conversion (see `date()`).

This ticket checks that all attendees have less than 8h of events for the starting day, otherwise an exception is thrown.

Side-note: it also breaks the tests related to events... it may be easier to wait for changes of events model.

### CODE REVIEW

**Comment by tkuy** [comment]
Please, respect the name convention for the branches : feature/TK-441-description-of-your-ticket

File: src/test/java/fr/lunatech/timekeeper/services/EventTemplateServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import javax.inject.Inject;
+import java.time.LocalDateTime;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+@QuarkusTest
+@QuarkusTestResource(H2DatabaseTestResource.class)
+@QuarkusTestResource(KeycloakTestResource.class)
+@DisabledIfEnvironmentVariable(named = "ENV", matches = "fast-test-only")
```
Comment (nicmarti) [inline src/test/java/fr/lunatech/timekeeper/services/EventTemplateServiceTest.java]: now should be @Tag("integration") cc @ani2fun

**Comment by nicmarti** [comment]
Transfered to @nicmarti as we need also to refactor EventTemplate andd remobe the list of attendees from the template

**Comment by nicmarti** [comment]
The EventTemplate should not store the list of Attendees. This is a design error, and we created another PR, that obsoletes this PR. 
Thus I will close it.

**Pull request closed without merging**